dae/dae/genomic_resources/tests/test_cli_browse.py

```diff
@@ -55,6 +55,10 @@ def test_cli_browse_with_grr_argument(repo_fixture, repo_def, capsys):
 
     assert err == ""
     assert out == \
+        "Working with GRR definition: " + str(repo_def) + "\n" \
```

Comment:
When constructing strings always prefer using f-strings. Don't use `+` for strings.